daemon/mgr/spec_hook.go

```diff
@@ -152,6 +158,24 @@ func setMountTab(ctx context.Context, c *Container, spec *SpecWrapper) error {
 	return nil
 }
 
+func setNvidiaHook(ctx context.Context, c *Container, spec *SpecWrapper) error {
+	n := c.HostConfig.NvidiaConfig
+	if n == nil {
+		return nil
+	}
+	path, err := exec.LookPath("nvidia-container-runtime-hook")
+	if err != nil {
+		return err
+	}
+	args := []string{path}
+	nvidiaPrestart := specs.Hook{
+		Path: path,
+		Args: append(args, "prestart"),
+	}
+	spec.s.Hooks.Prestart = append(spec.s.Hooks.Prestart, nvidiaPrestart)
```

Comment:
If we already has a prestart hook, will the nvidia prestart take effect here? Just double check this.

Reply:
Sure, prestart commands are executed after the container namespaces are created.